utils/prezly/context.ts

```diff
@@ -0,0 +1,13 @@
+import { Category, Newsroom } from '@prezly/sdk/dist/types';
+import { createContext } from 'react';
+
+interface INewsroomContext {
+    newsroom: Newsroom | null;
+    categories: Array<Category>;
+    selectedCategory?: Category;
+}
+
+export const NewsroomContext = createContext<INewsroomContext>({
+    newsroom: null,
+    categories: [],
+});
```

Comment:
What do you think about additional proxying component?

```suggestion
});

const NewsroomContextProvider: FunctionComponent<INewsroomContext> = ({
    children,
    categories,
    newsroom,
    selectedCategory
}) => (
    <NewsroomContext.Provider value={{ categories, newsroom, selectedCategory }}>
        {children}
    </NewsroomContext.Provider>
);
```

Usage will be a bit nicer then:
```
<NewsroomContextProvider newsroom={newsroom} categories={categories}>...
```

Reply:
actually usage seems a bit less nicer because you have to write each name twice 😅